com.unity.shadergraph/Documentation~/Refract-Node.md

```diff
@@ -0,0 +1,42 @@
+# Refract Node
+
+## Description
+
+Generates a refraction [Ibn Sahl-Snell-Descartes's Law](https://en.wikipedia.org/wiki/Snell%27s_law). Based on an **Incident** vector, the **Normal** of the surface and **Eta** we produce the refracted vector.
+
+Based on Ibn Sahl's Law, for a given refractive index of the medium, we can reach an angle where all the surface behave like a perfect mirror, that's why in **Safe** Mode generate a null vector, this implementation is here to not double the reflection intensity which is handle by Fresnel.
+
+## Ports
+
+| Name        | Direction           | Type  | Binding | Description |
+|:------------ |:-------------|:-----|:---|:---|
+| Incident      | Input | Vector | None | Incident normalied vector to the surface (From surface to source, for instance from pixel to eye). |
+| Normal      | Input | Vector | None | Normal normalized of the surface |
```

Comment:
Normal normalized what of the surface? 

I'm not sure what you mean.

Reply:
A vector can have a norm/length larger than one, here I underline the fact that vector should be normalized.
Like the node View Vector/View Direction, one is normalized and the other one not.

Reply:
> The normalized normal of the surface that should cause the refraction. 

Please change the text to the above.